CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+### Added
```

Comment:
Don't forget the header here. :slightly_smiling_face:


```suggestion
## [Unreleased]

### Added
```

The link is already defined at the bottom of the file:
```
[Unreleased]: https://github.com/gonzalo-bulnes/simple_token_authentication/compare/v1.16.0...master
```

Reply:
Sure, thanks for the hint. I've updated the changelog. Best wishes!